go.mod

```diff
@@ -4,6 +4,7 @@ go 1.14
 
 require (
 	code.rocketnine.space/tslocum/cview v1.5.4
+	github.com/atotto/clipboard v0.1.4 // indirect
```

Comment:
Make sure to run `go mod tidy` to get dependencies in order. Running that should remove the `// indirect` part of this line.

Reply:
I did wonder about the naming. Now updated to match the existing naming convention. I think I caught all of them.

Hopefully, I am now capturing the error in `display/display.go` in an appropriate way.

Reply:
This part looks good. I will review the rest and let you know.